### PR TITLE
Feature/optional descriptions

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: VotingApplication/VotingApplication.Web/Views/Routes/ManageOptions.cshtml [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+                    </span>
+                    <span class="icon-padded-left" ng-if="editOptionToggle"><input ng-model="option.Description" /></span>
                     @*Angular requires a button for submission. Go figure.*@
                     <button type="submit" hidden></button>
                 </div>
             </form>
         </div>
-        <div class="manage-section">
-            <form name="newOptionForm" ng-submit="add(newOptionForm);clear(newOptionForm);updatePoll();">
+        <div class="manage-section highlighted">
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Views/Routes/ManageOptions.cshtml]: I'm not sure the highlighting is necessary, might be worth checking it with Dave

**Comment by JDawes-ScottLogic** [comment]
:+1:

**Comment by tpkelly** [comment]
Looks good to me, +1

**Comment by stevenhillcox** [comment]
If you are both OK with this, can one of you merge it in please.

**Comment by JDawes-ScottLogic** [comment]
:+1: :shipit:

**Pull request merged**